sdk/Sdk.Generators/FunctionMetadataProviderGenerator/FunctionMetadataProviderGenerator.Parser.cs

```diff
@@ -573,7 +573,7 @@ private bool TryCreateBindingDict(AttributeData bindingAttrData, string bindingN
                     }
                     else
                     {
-                        bindings[propertyName] = prop.Value!.ToString();
+                        bindings[propertyName] = prop.Value!;
```

Comment:
Is line 562 using the correct casing?

Reply:
Yes, see the other comment for `SupportsDeferredBinding`.